Shared/Strings/Strings.swift

```diff
@@ -28,6 +28,8 @@ internal enum L10n {
   internal static let allGenres = L10n.tr("Localizable", "allGenres", fallback: "All Genres")
   /// All Media
   internal static let allMedia = L10n.tr("Localizable", "allMedia", fallback: "All Media")
+  /// Lettered Scrollbar
+  internal static let alphaPickerTitle = L10n.tr("Localizable", "alphaPickerTitle", fallback: "Lettered Scrollbar")
```

Comment:
Agreed. I think AlphaPicker makes sense as the "behind the scenes" name since it matches Jellyfin-Web but the only reference to "Lettered Scrollbar" should be the String. So changing that should be easy so long as we're fine with AlphaPicker as the variable name?

Any thoughts on other names? Lettered Scrollbar was my Hail Mary because I'm drawing a blank...

Reply:
Temporary change to Letter Slicer??? I dunno, kind of does the same thing as an Excel Slicer? But that's just where my head is today... 100% open to change that naming but I'll need someone a little more creative to provide a good name.